clang/test/CodeGen/aie/address_space/address-space-ctor.cpp

```diff
@@ -0,0 +1,30 @@
+//===- address-space-ctor.cpp -----------------------------------*- C++ -*-===//
+//
+//  Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+//  See https://llvm.org/LICENSE.txt for license information.
+//  SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+// Modifications (c) Copyright 2024 Advanced Micro Devices, Inc. or its affiliates
+//
+//===----------------------------------------------------------------------===//
+
+// RUN: %clang_cc1 -triple aie %s -std=c++14 -triple=spir -verify -fsyntax-only
```

Comment:
Triple is twice here (spir). For now I think it can work with the default triple, but as you plan to extend `isAddressSpaceSupersetOf`, you can keep as aie/aie2.